bat/tests/mixin-repo-commands/run.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "Verify clear repo is unable to be removed by mixin" {
+  run sudo mixin repo remove clear
+
+  [[ "$status" -eq 1 ]]
+  [[ "$output" =~ "The clear repo is mandatory and cannot be removed" ]]
+  [[ ! "$output" =~ "Removed clear repo." ]]
```

Comment:
I don't think there will ever be a case where the line above passes and this one fails, but on the other hand I see no harm in having them both here, so this is ok.